Clearer billing language.

julianeon opened the pull request:
Made the spacing slightly clearer. Added a "prices are impacted by" paragraph and a "stopping charges" section.

rohit2b commented on content/billing.md:
```diff
@@ -26,12 +32,21 @@ Galaxy Prepaid Pricing guarantees a specific amount of container capacity at a d
+Every month a statement is emailed for your total monthly usage. Statements show metered usage broken out by app. You may view and download past statements in your Galaxy account settings.
+
+<h2 id="stopping-charges">Stopping Charges</h2>
+While you can't stop charges for resources used in the past, you can stop charges, going forward, by stopping or deleting your containers.
```
should be "by stopping or deleting you apps".

julianeon commented:
I changed the language to say "you can stop charges by stopping or deleting your apps," as requested, but incorrectly committed that git message (not the change itself). However, that change has been made. I also added a space for better spacing after taking a closer look in hexo server; that is noted in the commit message.

rohit2b commented:
looks good, thanks @julianeon !
